torch/optim/lr_scheduler.py

```diff
@@ -163,6 +163,12 @@ def step(self, epoch=None):
         self._last_lr = [group['lr'] for group in self.optimizer.param_groups]
 
 
+# Including _LRScheduler for backwards compatibility
+# Subclass instead of assign because we want __name__ of _LRScheduler to be _LRScheduler (assigning would make it LRScheduler).
+class _LRScheduler(LRScheduler):
+    pass
```

Comment:
If _LRScheduler was private in the first place, do we actually need this?

Reply:
I see a good amount of internal and external usage that would break.

Reply:
Right, I was surprised that it could still be imported even though it was marked with _